tree/ntuple/src/RField.cxx

```diff
@@ -860,7 +860,8 @@ void ROOT::RNullableField::GenerateColumns()
 
 void ROOT::RNullableField::GenerateColumns(const ROOT::RNTupleDescriptor &desc)
 {
-   GenerateColumnsImpl<ROOT::Internal::RColumnIndex>(desc);
+   if (!fIsEvolvedFromInnerType)
```

Comment:
Maybe add `R__ASSERT(!fIsEvolvedFromInnerType)` in `RNullableField::GenerateColumns` (as we (will) have in `RField<std::vector<bool>>::GenerateColumns()`)?